### PR TITLE
PM-171 remove confirmation alerts for restoring an item

### PR DESCRIPTION
**Pull request by Jingo88**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ X ] Other
```

## Objective

Removing the confirmation alert for restoring an item from Trash. Keep toasts

Confirmation alerts should only be used as warnings if something is being deleted or for irreversible actions. Restoring an item does not fall under this definition.

## Code changes

Removed confirmation alert lines from 4 files. 

## Screenshots

Below are clips showing the behavior in Web, Browser, and Desktop

https://github.com/bitwarden/clients/assets/8302660/fc65838f-4ea4-4526-861c-d4395f629983

https://github.com/bitwarden/clients/assets/8302660/85c90c5a-a57e-4708-969a-04b0f23d4b02

https://github.com/bitwarden/clients/assets/8302660/5b58dbeb-4cd8-4272-9eab-bb9a2068cc5c

Removed Bulk Restore Dialog confirmation in individual and org vault

https://github.com/bitwarden/clients/assets/8302660/9d5fb657-b977-4ba2-a74c-8e5c2d09a85a

### CODE REVIEW

File: libs/angular/src/vault/components/view.component.ts [hunk 1]
```diff
@@ -209,16 +209,6 @@ export class ViewComponent implements OnDestroy, OnInit {
       return false;
     }
 
-    const confirmed = await this.dialogService.openSimpleDialog({
-      title: { key: "restoreItem" },
-      content: { key: "restoreItemConfirmation" },
```
Comment (differsthecat) [inline libs/angular/src/vault/components/view.component.ts:214]: Do we need these translations anymore? We can delete them if so 🙂

**Review by differsthecat — COMMENTED** [review]
Looks good! One small comment

**Review by differsthecat — DISMISSED** [review]
Looks, good, thanks @Jingo88 !

**Comment by Jingo88** [comment]
Updated PR with commit removing the bulk-restore-dialog confirmation. Video added to PR description.

**Review by differsthecat — APPROVED** [review]
Looking good, thanks @Jingo88 !

**Pull request merged**